AccountTrigger.trigger

```diff
@@ -1,12 +1,18 @@
 /*
 Trigger for demo purpose
 */
-trigger AccountTrigger on Account(Before inser, After insert){
+trigger AccountTrigger on Account(Before insert, After insert){
 
 
 for(Account acc: Trigger.New){
 
-    //logic goes for sales-account new feature
+   Contact conList = [Select Id, name from contact where AccounId =: acd.id];
```

Comment:
best practice is not followed

Set<Id> ids = new <set>{};
for(Account acc: Trigger.New){
ids.add(acc.id);
}

......


...